apps/mobile/src/hooks/use-migrate-current-account.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+import { useAccounts } from '@/store/accounts/accounts.read';
+import { useSettings } from '@/store/settings/settings';
+
+// This hook is temporary
```

Comment:
The hook is here to stay but we need to set the data temporarily? 🤔 

So if users update, and we don't have a `currentAccount` selected, this pre-sets it for them?